src/components/reference-book/exercise.cjsx

```diff
@@ -14,7 +14,7 @@ ReferenceBookExercise = React.createClass
   displayName: 'ReferenceBookExercise'
   render: ->
     {exerciseAPIUrl} = @props
-    {items} = ReferenceBookExerciseStore.get(exerciseAPIUrl)?
+    {items} = ReferenceBookExerciseStore.get(exerciseAPIUrl)
```

Comment:
Wow. I would not have expected a single question mark would cause so much trouble.